src/pcapi/utils/image_conversion.py

```diff
@@ -18,6 +19,15 @@ def standardize_image(image: bytes, crop_params: tuple = None) -> bytes:
     return standard_image
 
 
+def standardize_image_v2(image: bytes, crop_params: tuple = None) -> bytes:
+    crop_params = crop_params or DO_NOT_CROP
+    raw_image = PIL.Image.open(io.BytesIO(image)).convert("RGB")
+    cropped_image = _crop_image_v2(crop_params[0], crop_params[1], crop_params[2], raw_image)
```

Comment:
peux-tu expliciter les variables pour indiquer d'où elles viennent ? 
```suggestion
   x_position = crop_params[0]
    cropped_image = _crop_image_v2(x_position, crop_params[1], crop_params[2], raw_image)
```
etc ?

Reply:
Oui, sinon ça te va si j'utilise l'affectation par destruction ?
```suggestion
    x_position, y_positionn, crop_size = crop_params
    cropped_image = _crop_image_v2(x_position, y_position, crop_size, raw_image)
```